cmd/main.go

```diff
@@ -98,6 +102,20 @@ func main() {
 				}
 			}
 
+			if cfg.Logger.Type != "" {
+				lo, err = logger.NewLogger(cfg.Logger)
```

Comment:
Where a logger is not provided, the system should default to console logger with log level -- info.